.github/workflows/on_push_protected_branches.yml

```diff
@@ -30,6 +30,11 @@ jobs:
       - 
         name: "Linter"
         run: make lint
+      -
+        name: "Send coverage"
+        uses: shogo82148/actions-goveralls@v1
+        with:
+          path-to-profile: coverage.out
```

Comment:
I guess we need to generate the coverage profile, when running tests, right? Compare: https://github.com/mattn/goveralls/blame/master/README.md#L65

Reply:
exactly and the coverage profile is being generated by make lint, see the changes on MAKEFILE.